app/experimenter/experiments/serializers/design.py

```diff
@@ -170,6 +170,20 @@ class Meta(ExperimentDesignVariantBaseSerializer.Meta):
         model = ExperimentVariant
 
     def validate_preferences(self, data):
+        variant = None
```

Comment:
`variant = self.model.objects.get(id=self.data.get('id'))`

Reply:
I tried this, but self doesn't have a model attribute, or so the error says. Another odd this is that self.data was coming in with `none` and blank fields, which was confusing. I think we should go with what I have.

Reply:
Ah it should be `self.Meta.model`, but yes, it works as is so land it 👍